customer-managed/aws/README.md

```diff
@@ -1,268 +1,115 @@
-This document contains customer instructions for early access to the Redpanda BYOVPC AWS feature, for proof of concept 
-purposes.
+# Overview
 
-# Prerequisites
-
-1. Your Redpanda representative must have enabled the BYOVPC feature flag for your organization.
-2. Familiarity with the [Redpanda Public API](https://redpanda-api-documentation.netlify.app/). (e.g. How to 
-authenticate and create a cluster using the API.)
-3. Access to an AWS project in which you’d like to create your cluster.
-4. [Minimum permissions](https://github.com/redpanda-data/cloud-examples/customer-managed/aws/terraform/iam_rpk_user.tf) in that 
-AWS project. (Note that the link to the minimum permissions is provided simply as documentation, we do not anticipate 
-that you would create the role in that file, it is provided only as a way to communicate the actions that will be 
-required of the employee running rpk byoc apply.)
-
-# Create VPC and other sensitive resources
-
-The [terraform code](https://github.com/redpanda-data/cloud-examples/customer-managed/aws/terraform) contains the 
-specifications for what resources are expected to exist and how they are expected to be configured.
+This repository contains [Terraform](https://developer.hashicorp.com/terraform) code that describes the resources
+customers are responsible for creating in association with a Redpanda Customer-Managed VPC cluster. These resources
+should be created in advance by the customer and then provided to Redpanda during cluster creation.
 
-This terraform code will create the following resources:
-* IAM Roles
-* IAM Instance Profiles
-* IAM Role Policy Attachments
-* IAM Policies
-* Elastic IP
-* Internet Gateway
-* Route
-* VPC
-* Subnets
-* VPC Endpoint
-* Route Tables
-* Route Table Associations
-* VPC Endpoint Route Table Associations
-* Security Groups
-* Security Group Rules
-* S3 Buckets
-* S3 Bucket Server-Side Encryption Configurations
-* S3 Bucket Versioning
-* S3 Bucket Ownership Controls
-* DynamoDB Table
+TODO: doc link here (would be nice to have an overview doc of Customer-Managed VPC that is agnostic of cloud provider)
+For more information on Redpanda's Customer-Managed VPC offering see...
```

Comment:
```suggestion
For more information on Redpanda's customer-managed VPC offering, see...
```
no link for this today, but you could link to this BYOC architecture? https://docs.redpanda.com/redpanda-cloud/get-started/cloud-overview/#byoc-architecture